WalkingChallenge/Assets.xcassets/Aga Khan Foundation LOGO_raster.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "Aga Khan Foundation LOGO_raster.png",
```

Comment:
I don't think that we need the `_raster` suffix to indicate that the image has been rasterized.

Reply:
probably not, this was just the name of the file when downloaded from UX source.